source/features/global-search-filters.tsx

```diff
@@ -1,54 +0,0 @@
-import React from 'dom-chef';
-import {XIcon} from '@primer/octicons-react';
-import elementReady from 'element-ready';
-import * as pageDetect from 'github-url-detection';
-
-import features from '../feature-manager';
-import SearchQuery from '../github-helpers/search-query';
-import {getUsername} from '../github-helpers';
-
-async function init(): Promise<void> {
-	const filters = [
-		['Forks', 'fork:true'],
-		['Private', 'is:private'],
```

Comment:
On the bottom left sidebar:

<img width="133" alt="Screenshot" src="https://user-images.githubusercontent.com/1402241/215421087-9015e108-1876-4955-adc7-ee6cef5e3c0f.png">